tests/tools/onert_train/src/args.h

```diff
@@ -55,9 +57,9 @@ class Args
   const int getEpoch(void) const { return _epoch; }
   const int getBatchSize(void) const { return _batch_size; }
   const float getLearningRate(void) const { return _learning_rate; }
-  const int getLossType(void) const { return _loss_type; }
-  const int getLossReductionType(void) const { return _loss_reduction_type; }
-  const int getOptimizerType(void) const { return _optimizer_type; }
+  const NNFW_TRAIN_LOSS getLossType(void) const { return _loss_type; }
+  const NNFW_TRAIN_LOSS_REDUCTION getLossReductionType(void) const { return _loss_reduction_type; }
+  const NNFW_TRAIN_OPTIMIZER getOptimizerType(void) const { return _optimizer_type; }
```

Comment:
(ouf of this draft's original purpose) 
But I'd like to suggest to use `NNFW_TRAIN_*` in `args.h`.

When one of train parameter extended, updating both main and args is a bit troblesome.
Also, I'd like to reduce main, by removing `convertLossType`, `convertLossReductionType` and `convertOptType`.